kafka/src/main/resources/config/base.conf

```diff
@@ -12,36 +12,64 @@ metric-router {
   streams {
     application.id = "metric-router"
   }
-  topic = "metrics"
+  inbound-topic = "metrics"
+
+  # TODO This doesn't do anything yet, but it will once MetricRouter maps the mpoints. [WLW]
+  outbound-topic = "mapped-metrics"
+}
+
+anomaly-detector-manager {
+  streams {
+    application.id = "anomaly-detector-manager"
+    default.value.serde = "com.expedia.adaptivealerting.kafka.serde.JsonPojoSerde"
```

Comment:
Seems like a serde is provided on the input and output (`jsonMpointSerde` and `jsonAnomalySerde`) so this probably isn't needed otherwise might need to specifiy `JsonPojoClass` as well?